### DISCUSSION TITLE
不能在线升级，一直提示已经是最新版本

### DISCUSSION THREAD

**Discussion by lichengwu** [1Panel-dev/1Panel#5496] [Q&A]
### 1Panel 版本

v1.10.16-lts

### 问题描述

<img width="877" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/1235434/53405afe-4ddf-4efc-af3b-27f0f1747685">

无法升级，每次都提醒最新版

### 重现步骤

<img width="877" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/1235434/53405afe-4ddf-4efc-af3b-27f0f1747685">

### 期待的正确结果

升级最新版本

### 附加信息

或者有没有什么手动升级办法，保障数据和服务不受影响

**Comment by wanghe-fit2cloud** [comment]
应该是由于网络原因，导致无法获取最新版本。可以进入日志审计页面查看系统日志，获取更多详细信息。

目前先参考这篇论坛文章，手动升级到最新版本。
最新版本在面板设置页面中支持设置代理服务器功能，可以考虑添加代理服务器来解决网络问题。
https://bbs.fit2cloud.com/t/topic/5579
